core/services/ocr2/plugins/functions/integration_tests/v1/functions_integration_test.go

```diff
@@ -22,13 +23,12 @@ var (
 func TestIntegration_Functions_MultipleV1Requests_Success(t *testing.T) {
 	t.Skip()
```

Comment:
keeping this for now

Reply:
Are planning to remove this once you confirm this PR works as expected?